src/main/resources/templates/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
```

Comment:
Added thymeleaf so you can use it to access your data later down the road. I needed to use thymeleaf to add the reusable navbar

Reply:
Nice! Thanks!